master/pkg/searcher/operations.go

```diff
@@ -144,14 +135,14 @@ func NewCreate(
 	}
 }
 
-// NewCreateFromCheckpoint initializes a new Create operation with a new request ID and the given
+// NewCreateFromParent initializes a new Create operation with a new request ID and the given
 // hyperparameters and checkpoint to initially load from.
-func NewCreateFromCheckpoint(
-	rand *nprand.State, s hparamSample, checkpoint Checkpoint,
+func NewCreateFromParent(
```

Comment:
q: does parent mean parent trial?

Reply:
ah, originally i changed the create to have a parent request ID instead of a checkpoint, but then i realized it caused issues (would need shim all the saved creates for PBT) so I decided not to do it.

Reply:
I mean, I'm still going to use the parent ID, I just decided to leave the checkpoint wrapper around it to save on breakage. I can change it and shim it also if you think we should.